sfa_dash/blueprints/reports.py

```diff
@@ -125,36 +121,113 @@ def post(self):
 class ReportView(BaseView):
     template = 'data/report.html'
 
+    def should_include_timeseries(self):
+        if self.metadata['status'] != 'complete':
+            return False
+        start_string = self.metadata['report_parameters'].get('start')
+        end_string = self.metadata['report_parameters'].get('end')
+        start = pd.Timestamp(start_string)
+        end = pd.Timestamp(end_string)
+        report_period = (end - start)
+        total_data_points = 0
+        raw_report = self.metadata['raw_report']
+        pfxobs = raw_report['processed_forecasts_observations']
+        for fxobs in pfxobs:
+            interval_length = pd.to_timedelta(fxobs['interval_length'],
+                                              unit='m')
+            fxobs_data_points = report_period / interval_length * 2
+            total_data_points = total_data_points + fxobs_data_points
+        return total_data_points < current_app.config['REPORT_DATA_LIMIT']
```

Comment:
so this could benefit from a count of the total valid points for each fx/obs stored in PreprocessingResult?

Reply:
and maybe any logic like this should go in core templates.py anyway? I don't think it's ever really useful to put the plots in if browser rendering will take ages.

Reply:
I think that leaving it up to the end user is somewhat valuable, and if we move this logic to core it should have some override flag. The current warning explains why we've omitted the timeseries, but If someone really wants to looks at a bokeh graph for a long period, I think we should allow them to download it.